### ISSUE TITLE
MyReports "which is" Drop-Down needs to be limited in Width

### ISSUE THREAD

**Issue by dartajax** [ilios/frontend#1139]
See the screen shot below ...

![image](https://cloud.githubusercontent.com/assets/7493331/11289078/598e602c-8ede-11e5-8c63-3c34227d3459.png)

There should be an indicator that this is a drop-down menu.  The reason is shown below ..

I couldn't show the drop-down but the following SOD topic is way too long and causes the whole drop-down to be far too wide and hence not indentifiable as a drop-down.

![image](https://cloud.githubusercontent.com/assets/7493331/11289124/c46adc04-8ede-11e5-848a-0d8f403d1b93.png)

In other words we need to word wrap after a certain point and not allow the drop-down to grow in width without restriction.

**Comment by jrjohnson** [comment]
Dropdown options cannot be line wrapped.  We can set a max width on the control and then truncate any options that are too long.  Will that work?  Are the first 75 (for example) characters unique enough to pick a topic from?

**Comment by saschaben** [comment]
If we need to truncate, then it will have to be less than 75 chars. 50 is more likely to fit.

I suspect that will provide enough differentiation for selection.
